spec/requests/bids_spec.rb

```diff
@@ -53,6 +54,10 @@
       it 'returns a 404 status code' do
         expect(status).to eq(404)
       end
+
+      it 'should not create a bid' do
+        expect(auction.bids.count).to eq(@bid_count)
```

Comment:
an added bonus for inlining the exercise is that it can be hard to see what is being tested when the exercise just happens once per file. Similar arg to not using `let` but I do think that having the *what is being tested* line exist within each test